src/atoms/Space/__tests__/__snapshots__/Space.web.test.js.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Renders <Space /> correctly should apply margin to child component 1`] = `<div />`;
```

Comment:
There are no styles visible for paddings you applied. Also add some text. It's just rendering View right now